ProcessMaker/processmaker
# UI improvements: search bar, element spacing, page width, notifications popover

## Changes
- Uses Bootstrap's flex utilities to implement new search bar design on the following pages:
  - Categories
  - Processes
  - Screens
  - Scripts
  - Environment variables
  - Users
  - Groups
  - Notifications
- Updates the search bar in the New Request modal to match new design
- Improves margins and padding on all list views throughout the app
- Improves margins and padding on vuetables, including those without pagination
- Converts the Task and Request detail screens to full-width
- Fixes a spacing issue with the notifications popover

## Testing Instructions
- Run `npm run dev` to generate the updated New Request modal
- Review each of the pages listed above
  - Attempt to search
  - Attempt to use any action buttons next to the search bar
- Run a process with tasks and ensure that the Task and Request detail screens work properly at full-width
- Run a process with notifications and view the Notifications popover to ensure it appears properly

## Examples

**Before**
![Screen Shot 2019-11-20 at 3 36 08 PM](https://user-images.githubusercontent.com/867714/69287576-94ae8580-0bab-11ea-8861-49361d59699e.png)

**After**
![Screen Shot 2019-11-20 at 3 35 36 PM](https://user-images.githubusercontent.com/867714/69287592-9bd59380-0bab-11ea-86fa-45a6478df3e1.png)

----

**Before**
![Screen Shot 2019-11-18 at 12 17 20 PM](https://user-images.githubusercontent.com/867714/69090934-1eb8eb80-09fe-11ea-94d5-47fca9079012.png)

**After**
![Screen Shot 2019-11-18 at 12 16 22 PM](https://user-images.githubusercontent.com/867714/69091057-5889f200-09fe-11ea-8a30-202dd6caae45.png)

----

**Before**
![Screen Shot 2019-11-20 at 3 39 04 PM](https://user-images.githubusercontent.com/867714/69287745-3a61f480-0bac-11ea-9417-2cde3f77d17e.png)

**After**
![Screen Shot 2019-11-20 at 3 38 27 PM](https://user-images.githubusercontent.com/867714/69287752-3fbf3f00-0bac-11ea-8f8a-7cd2d7f5b148.png)

----

**Before (Small Screen Size)**
![Screen Shot 2019-11-18 at 12 35 32 PM](https://user-images.githubusercontent.com/867714/69091854-0649d080-0a00-11ea-80e8-9f644a7a9d9b.png)

**After (Small Screen Size)**
![Screen Shot 2019-11-18 at 12 36 08 PM](https://user-images.githubusercontent.com/867714/69091870-0d70de80-0a00-11ea-8df2-bb7e16bf92e3.png)

----

**Before**
![Screen Shot 2019-11-20 at 3 39 24 PM](https://user-images.githubusercontent.com/867714/69287787-5c5b7700-0bac-11ea-8369-4b649a8be792.png)

**After**
![Screen Shot 2019-11-20 at 3 40 18 PM](https://user-images.githubusercontent.com/867714/69287798-61b8c180-0bac-11ea-9888-8d003ee81a24.png)

Closes #2569.
